ibodata/lateral_profile.py

```diff
@@ -6,56 +6,107 @@
 
 
 class LateralProfile(Profile):
-    def penumbra_left(self):
+    def penumbra_right(self):
         """
-        In case of good data returns positive number
+        In case of good data returns float
```

Comment:
It is always float ? What if profile contains integer numbers, like:
`X = [0,1,2,3]`  and `Y=[0,1,10,1]`

Better use floating-point number than float